{"metadata":{"modules":[{"name":"Core"}],"role":"collectionGroup","title":"Error Implementations"},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"article","topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/Core\/documentation\/Core\/NetworkingError\/localizedDescription"],"generated":true}],"hierarchy":{"paths":[["doc:\/\/Core\/documentation\/Core","doc:\/\/Core\/documentation\/Core\/NetworkingError"]]},"sections":[],"identifier":{"url":"doc:\/\/Core\/documentation\/Core\/NetworkingError\/Error-Implementations","interfaceLanguage":"swift"},"variants":[{"paths":["\/documentation\/core\/networkingerror\/error-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://Core/documentation/Core/NetworkingError/localizedDescription":{"type":"topic","role":"symbol","title":"localizedDescription","kind":"symbol","url":"\/documentation\/core\/networkingerror\/localizeddescription","abstract":[],"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"identifier":"doc:\/\/Core\/documentation\/Core\/NetworkingError\/localizedDescription"},"doc://Core/documentation/Core/NetworkingError":{"type":"topic","title":"NetworkingError","abstract":[{"type":"text","text":"The error about the networking."}],"kind":"symbol","identifier":"doc:\/\/Core\/documentation\/Core\/NetworkingError","url":"\/documentation\/core\/networkingerror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkingError"}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"NetworkingError"}]},"doc://Core/documentation/Core":{"url":"\/documentation\/core","kind":"symbol","role":"collection","abstract":[],"type":"topic","identifier":"doc:\/\/Core\/documentation\/Core","title":"Core"}}}